tests/web/test_transactions.py

```diff
@@ -12,18 +12,15 @@
 TX_ENDPOINT = '/api/v1/transactions/'
 
 
-@pytest.mark.bdb
-@pytest.mark.usefixtures('inputs')
-def test_get_transaction_endpoint(b, client, user_pk):
-    input_tx = b.get_owned_ids(user_pk).pop()
-    tx = b.get_transaction(input_tx.txid)
-    res = client.get(TX_ENDPOINT + tx.id)
-    assert tx.to_dict() == res.json
+pytestmark = pytest.mark.abci
```

Comment:
Why do those tests need this marker? /cc @kansi

Reply:
I would also like to understand why every test in this module needs this marker?

Reply:
All of the tests in this module are API-level. Only a few of them do not require the running Tendermint - I was thinking it would not hurt to mark the whole module as such.